homework/src/index.js

```diff
@@ -33,14 +33,123 @@
   function main(url) {
     fetchJSON(url, (err, data) => {
```

Comment:
If you know that the data you get is a list of repositories, a better name for `data` would be `repositories`.